roles/KubevirtCommonTemplatesBundle/tasks/main.yml

```diff
@@ -41,6 +84,23 @@
     namespace: "{{ meta.namespace }}"
     conditions:
     - type: Available
-      status: "{{ 'True' if new_templates <= deployed_templates_after else 'False' }}"
+      status: "{{ 'True' if available else 'False' }}"
       reason: "available"
-      message: "Common templates available."
+      message: "Common templates available (deployed templates: {{ deployed_templates_after }}, desired deployed templated: {{ new_templates }})."
+
+- name: Set degraded condition
```

Comment:
Same here, how can this happen? The playbook installs all of them directly..

Reply:
I set this condition in favor of HCO, do you think it would be better to place a constant value for the status? Because you are correct, once the playbook reaches this point all templates should exist already and the condition status would never be true

Reply:
Yes, use a constant. I think it even was like that in the past.

Reply:
Done